src/vs/workbench/api/node/extHostLanguageFeatures.ts

```diff
@@ -214,6 +214,33 @@ class ReferenceAdapter {
 	}
 }
 
+class ExternalReferenceAdapter {
+
+	constructor(private _provider: vscode.ExternalReferenceProvider) {
+	}
+
+	provideExternalReferences(query: modes.ISymbolDescriptor, hints: { [hint: string]: any }, progress: (references: modes.IReferenceInformation[]) => void): TPromise<modes.IReferenceInformation[]> {
+		const convertReferences = (references: vscode.ReferenceInformation[]) => {
```

Comment:
Yes, totally agree with consistency over style :)

Regarding rebasing: This isn't a big difference and unrelated to the discussion, but I wouldn't rebase on upstream VS Code's master. That would rewrite our history, we would lose the ability to track when a change was made. For a long-living branch like this let's _merge_ VS Code's upstream with proper merge commits so the commit history is maintained